wca-regulations.md

```diff
@@ -192,6 +192,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 5b5c) If more than one part with one colored face is affected, the puzzle is considered unsolved (DNF).
         - 5b5d) If one or more parts with more than one colored face are affected, the puzzle is considered unsolved (DNF).
         - 5b5e) Regulations [5b5c](regulations:regulation:5b5c) and [5b5d](regulations:regulation:5b5d) supersede [5b5a](regulations:regulation:5b5a) and [5b5b](regulations:regulation:5b5b).
+    - 5b6) Definition: A part is a subset of puzzle components (in particular, any individual piece, cap, sticker, etc.).
```

Comment:
This looks good, but we don't use "in particular". I'd go for this:
```suggestion
    - 5b6) Definition: A part is a subset of puzzle components that can physically hold together (e.g. an individual piece, cap, single sticker, or sticker attached to another part).
```
I think we should also have an example like "2/3 of a corner", but I can't think of a good way to phrase that.